readme.md

```diff
@@ -1,72 +1,32 @@
 # Retrospect - Public GIT Repository #
-[![License](https://img.shields.io/badge/license-cc_by--nc--nd-brightgreen)](https://github.com/retrospect-addon/plugin.video.retrospect/blob/master/plugin.video.retrospect/license.txt)
+[![GitHub release (latest by date)](https://img.shields.io/github/v/release/retrospect-addon/plugin.video.retrospect)](https://github.com/retrospect-addon/plugin.video.retrospect/releases)
 [![Build Status](https://travis-ci.com/retrospect-addon/plugin.video.retrospect.svg?branch=master)](https://travis-ci.com/retrospect-addon/plugin.video.retrospect)
 [![Quality Gate Status](https://sonarcloud.io/api/project_badges/measure?project=retrospect-addon:plugin.video.retrospect&metric=alert_status)](https://sonarcloud.io/dashboard?id=retrospect-addon:plugin.video.retrospect)
-
+[![License](https://img.shields.io/badge/license-cc_by--nc--nd-brightgreen)](https://github.com/retrospect-addon/plugin.video.retrospect/blob/master/plugin.video.retrospect/license.txt)
+[![Python](https://img.shields.io/badge/python-2.7%20%7C%203.6-blue)](https://kodi.tv/article/attention-addon-developers-migration-python-3)
 
 This repository holds the main code for Retrospect. For more information of bug reporting please visit https://github.com/retrospect-addon/plugin.video.retrospect or https://www.rieter.net/content/.
 
-# Installing and updating Retrospect under *Kodi Leia* and up
-If you are using Kodi Leia then the installation is straight forward: you can install Retrospect and the dependencies from the Kodi repository. You will find Retrospect in the repository under the Video Add-ons. 
-Updating Retrospect will then automatically be done using the Kodi repository.
-
 # Installing Retrospect #
-There are a couple of ways to install and/or update Retrospect using this GIT repository:
-
-### 1 - Installing and updating Retrospect the 'Easy way' ###
-If a `plugin.video.retrospect-x.x.x.zip` is available from the download section, this zip can be installed using Kodi's _Install from ZIP_ feature. Keep in mind that you these zip files may not always be up-to-date.
-
-This method can also be used to install new versions of Retrospect and thus upgrading older installs.  
-
-### 2 - Installation of Retrospect the 'Advanced way' ###
-The installation comes down to putting the folders from the GIT repo (either via a _'GIT Clone'_ or _'Full Zip Download'_)in the Kodi add-on folder (very often: /home/<user>/.kodi/addons or c:\users\<user>\AppData\Roaming\Kodi\addons). The result should be that the *addons* folder now contains at least these folders:
-
```

Comment:
### 2 - Installation of Retrospect by using Github - the 'Advanced way' ###
This method comes down to putting the folders from the GIT repo (either via a _'GIT Clone'_ or _'Full Zip Download'_)in the Kodi add-on folder (by default: /home/<user>/.kodi/addons or c:\users\<user>\AppData\Roaming\Kodi\addons). This should result in the *addons* folder now containing at least these folders:

Reply:
I think you are reviewing the **old** versions! The red stuff is already removed